proto/osmosis/incentives/gauge.proto

```diff
@@ -50,6 +50,24 @@ message Gauge {
   ];
 }
 
+// SplittingPolicy determines the way we want to split incentives in groupGauges
+enum SplittingPolicy {
+  option (gogoproto.goproto_enum_prefix) = false;
+
+  Volume = 0;
+  Liquidity = 1;
+  Evenly = 2;
```

Comment:
```suggestion
  Evenly = 1;
```

Reply:
We're likely to simply remove liquidity based splitting for the reasons described in #6093